src/bioregistry/data/bioregistry.json

```diff
@@ -67765,6 +67765,38 @@
     "name": "Novus Biologicals",
     "uri_format": "https://www.novusbio.com/products/$1"
   },
+  "noyo": {
+    "contact": {
+      "email": "paul.fabry@usherbrooke.ca",
+      "github": "pfabry",
+      "name": "Paul Fabry",
+      "orcid": "0000-0002-3336-2476"
+    },
+    "contributor": {
+      "email": "paul.fabry@usherbrooke.ca",
+      "github": "pfabry",
+      "name": "Paul Fabry",
+      "orcid": "0000-0002-3336-2476"
+    },
+    "description": "NOYO is an ontology including all core classes and properties used in downstream ontologies (such as HOSO, HEPRO or PDRO for example) that constitues the ontological model of clinical information  from GRIIS Learning Health System platform.",
```

Comment:
A few ways to improve the description:
- It's not obvious from this where the acronym NOYO comes from
- mentions of other resources could be linked with markdown (at least written out in full)
- most importantly, I don't think that this description gives enough context as to what this is and what's going on other than mention of clinical information. How is this related to other resources that exist? Why should I use this semantic space instead of others? What is GRIIS learning health system platform?

Reply:
@pfabry just in case you didn't get a ping on this